routes/chatbot.ts

```diff
@@ -48,6 +49,23 @@ export async function initialize () {
 
 void initialize()
 
+async function processQueryWithChatGPT (user: User, req: Request, res: Response, next: NextFunction) {
+  const configuration = new Configuration({
+    apiKey: process.env.OPENAI_API_KEY,
+  })
+  const openai = new OpenAIApi(configuration)
+
+  const completion = await openai.createCompletion({
+    model: "text-davinci-003",
+    prompt: `Hello world ${user.email}`,
+  })
```

Comment:
🚫 **[Bearer]** <[javascript_third_parties_openai](https://docs.bearer.com/reference/rules/javascript_third_parties_openai)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>
# Sensitive data sent to OpenAI detected.
## Description
Leaking sensitive data to third-party is a common cause of data leaks and can lead to data breaches. This rule looks for instances of sensitive data sent to OpenAI.

## Remediations

When using a third-party, ensure all sensitive data is removed.